dvc/commands/imp_url.py

```diff
@@ -72,6 +73,13 @@ def add_parser(subparsers, parent_parser):
         default=False,
         help="Only create .dvc file without actually downloading it.",
     )
+    import_parser.add_argument(
+        "--no-download",
+        action="store_true",
+        default=False,
+        help="Create .dvc file with remote checksums without actually "
+        "downloading the associated data.",
```

Comment:
The top `help` is for `--no-exec`:
```suggestion
        help="Only create .dvc file "
        "without actually importing the target data.",
    )
    import_parser.add_argument(
        "--no-download",
        action="store_true",
        default=False,
        help="Create .dvc file including hash value(s) of the target data "
        "but don't actually download the file(s).",
```